src/tensorwaves/optimizer/scipy.py

```diff
@@ -0,0 +1,105 @@
+# cspell:ignore BFGS disp nfev
+"""Adapter to the `scipy.optimize` package."""
+
+import logging
+import time
+from datetime import datetime
+from typing import Any, Dict, Iterable, Mapping, Optional, Union
+
+from scipy.optimize import minimize
+from tqdm.auto import tqdm
+
+from tensorwaves.interfaces import Estimator, Optimizer
+
+from ._parameter import ParameterFlattener
+from .callbacks import Callback, CallbackList
+
+
+class ScipyMinimizer(Optimizer):
+    """The Scipy Optimizer adapter.
+
+    Implements the `~.interfaces.Optimizer` interface.
+    """
+
+    def __init__(
+        self,
+        method: str = "BFGS",
+        callback: Optional[Callback] = None,
+        use_analytic_gradient: bool = False,
+        **scipy_options: Dict[Any, Any],
+    ) -> None:
+        self.__callback: Callback = CallbackList([])
+        if callback is not None:
+            self.__callback = callback
+        self.__use_gradient = use_analytic_gradient
+        self.__method = method
+        self.__minimize_options = scipy_options
+        # self.__minimize_options.update({"disp": True})
```

Comment:
Can this be removed?

Reply:
Ah true i wanted to check if this is good default behaviour...

Reply:
db932dd659f4f383cf464643b0b36c6482fd0563